cmd/kube-proxy/app/server.go

```diff
@@ -941,3 +944,33 @@ func detectNodeIPs(client clientset.Interface, hostname, bindAddress string) (v1
 		}
 	}
 }
+
+// getNodeIP returns IPs for the node with the provided name.  If
+// required, it will wait for the node to be created.
+func GetNodeIPs(client clientset.Interface, name string) []net.IP {
```

Comment:
do we want to export this function? why don't we do the other way around and make this function private?
```suggestion
// getNodeIP returns IPs for the node with the provided name.  If
// required, it will wait for the node to be created.
func getNodeIPs(client clientset.Interface, name string) []net.IP {
```

Reply:
Yes, it's used in some "hollow proxier"